waspc/test/JsImportTest.hs

```diff
@@ -0,0 +1,53 @@
+module JsImportTest where
+
+import StrongPath (Dir', File', Path, Posix, Rel, (</>))
+import qualified StrongPath as SP
+import Test.Tasty.Hspec (Spec, describe, it, shouldBe)
+import Wasp.JsImport
+
+spec_JsImportTest :: Spec
+spec_JsImportTest = do
+  describe "makeJsImport" $ do
+    it "makes JsImport with default import from a path" $ do
+      makeJsImport pathToJsFile (JsImportModule "test")
+        `shouldBe` JsImport
+          { _path = pathToJsFile,
+            _name = JsImportModule "test",
+            _importAlias = Nothing
+          }
+  describe "applyJsImportAlias" $ do
+    it "applies alias to JsImport" $ do
+      applyJsImportAlias
+        (Just "alias")
+        (makeJsImport pathToJsFile (JsImportModule "test"))
+        `shouldBe` JsImport
+          { _path = pathToJsFile,
+            _name = JsImportModule "test",
+            _importAlias = Just "alias"
+          }
+  describe "generateJsImportStatement" $ do
```

Comment:
I think there are some opportunities to make the tests below easier to read / more maintainable. Some ideas:
1. All `it`s share same prefix: `generates import statement` -> that should be extracted into common `describe`.
2. Extracting some more of common logic. You did some of it with `pathToJsFile`, but you stopped there and I think you can go a bit further, I commented more on it below.

Maybe there are some other opportunities also.
We shouldn't overdo it, but still, testing code should also be subject to most of the rules we use in the normal code, which means keeping it DRY and SOLID and whatever other cool acronyms :D: in other words, keeping it clean. I think it can be ok to sometimes do it a bit less than we would do in the tested code though, but not too much.
The thing is, test code also needs to be maintained and readable -> if it is not, as a result you get that people don't want to change normal code because they don't want to deal with test code, or it stops being clear from test code what was actually the behaviour you wanted to test, ... .